spec/requests/cloud_object_store_containers_spec.rb

```diff
@@ -45,4 +47,47 @@
       expect(response.parsed_body).to include(expected)
     end
   end
+
+  context 'POST /api/cloud_object_store_containers/' do
+    it 'it can create cloud object store containers through POST' do
+      zone = FactoryBot.create(:zone)
+      provider = FactoryBot.create(:ems_storage, :zone => zone)
+      cloud_tenant = FactoryBot.create(:cloud_tenant, :ext_management_system => provider)
+
+      api_basic_authorize collection_action_identifier(:cloud_object_store_containers, :create, :post)
+      submit_data = {
+        :ems_id          => provider.id,
+        :name            => 'foo',
+        :emsType         => provider.type,
```

Comment:
@GilbertCherrie I just notice this in the submit data, which I don't think is correct.  cc @agrare @kbrock

Reply:
Yeah :ems_type looks correct I think, I can make this change if needed